include/openenclave/bits/eeid.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+/**
+ * @file eeid.h
+ *
+ * This file defines the EEID structure.
+ *
+ */
+
+#ifndef _OE_BITS_EEID_H
+#define _OE_BITS_EEID_H
+
+#include "properties.h"
+
+OE_EXTERNC_BEGIN
+
+#ifdef OE_WITH_EXPERIMENTAL_EEID
+/*
+**==============================================================================
+**
+** oe_eeid_t
+**
+**==============================================================================
+*/
+typedef struct oe_eeid_t_
+{
+    struct
+    {
+        uint32_t H[8];
+        uint32_t N[2];
+    } hash_state; /* internal state of the hash computation at the end of
+                           the enclave base image */
+    uint64_t signature_size; /* size of signature */
```

Comment:
What's the point of having a size variable when the array length is fixed to 1808?

Reply:
That's what we suggested in the design document for the platform-independent EEID. This way the structure has the same size and layout as the proposed one, but the current implementation is SGX-only. We could make it point into a new block at the end of `data` of course.